src/main/java/fi/vm/yti/terminology/api/importapi/ImportService.java

```diff
@@ -223,12 +223,39 @@ ResponseEntity<String> handleNtrfDocumentAsync(String format, UUID vocabularyId,
             System.out.println("Incoming transform error=" + ioe);
         } catch (XMLStreamException se) {
             System.out.println("Incoming transform error=" + se);
-        } catch(JAXBException je){
+        } catch (JAXBException je){
             System.out.println("Incoming transform error=" + je);
         }
         return new ResponseEntity<>( rv, HttpStatus.OK);
     }
 
+    public UUID handleSimpleExcelImport(UUID terminologyId, InputStream is) throws NullPointerException, IOException {
+        boolean exists = terminologyExists(terminologyId);
+        if (!exists) {
+            throw new NullPointerException("Terminology doesnt exist");
+        }
+
+        var graph = termedService.getGraph(terminologyId);
+        List<String> languages = graph.getProperties().get("prefLabel").stream().map(Property::getLang).collect(Collectors.toList());
```

Comment:
Get languages through prefLabel. This might cause a problem with old terminologies since it is a new requirement that prefLabel has to be defined in all languages

Reply:
I'd prefer method `getVocaboary` in TermedService. It seems that `getGraph` returns prefLabel in only one language

Reply:
Changed to use getVocabulary instead